build.gradle

```diff
@@ -246,6 +246,7 @@ task fastbuild(dependsOn: [
 
 task travisbuild(dependsOn: [
     fastbuild,
+    ':dbfit-java:db2:integrationTest',
```

Comment:
It's the total time until the first test is done. It's a speculative guess so far - I don't really know whether db2 is slower or not. Maybe the order is just fine as it is.